### PR TITLE
Enable any of rules for routing

### PR DESCRIPTION
**Pull request by heofs**
### What is the context of this PR?
This PR enables a user to route by using Any of (OR) or All of (AND) routing rules.

### How to review 
**author**
1. Create a new questionnaire
1. Fill in a question title and create a new answer (radio, number or currency).
1. Go to routing and add two new expressions to the rule.
1. Switch between "Match All of" and "Match Any of" and see that the AND and OR labels change on the left hand side.

**author-api**
* Check that the migration is okay

**publisher**
...

### CODE REVIEW

**Comment by heofs** [comment]
<https://trello.com/c/Pib1MNWh/463-enabling-any-of-rules-for-routing-on-radios-and-numbers-12>

**Comment by heofs** [comment]
https://trello.com/c/Pib1MNWh/463-enabling-any-of-rules-for-routing-on-radios-and-numbers-12

**Comment by heofs** [comment]
[Enabling 'Any of' rules for routing on radios and numbers (12)](https://trello.com/c/Pib1MNWh/463-enabling-any-of-rules-for-routing-on-radios-and-numbers-12)

File: eq-author-api/migrations/addExpressionOperator.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+//This is an auto-generated file.  Do NOT modify the method signature.
+
+module.exports = function addExpressionOperator(questionnaire) {
+  questionnaire.sections.map(section => {
+    section.pages.map(page => {
+      if (page.routing) {
+        page.routing.rules.map(rule => {
+          if (!rule.expressionGroup.operator) {
+            rule.expressionGroup.operator = "And";
```
Comment (pricea1) [inline eq-author-api/migrations/addExpressionOperator.js]: Should this refer to the AND constant?

Comment (heofs) [reply eq-author-api/migrations/addExpressionOperator.js]: There is only a constant for it on the publisher side atm, but I will add one for API and frontend.

File: eq-author-api/src/businessLogic/createRoutingRule.js [hunk 2]
```diff
@@ -6,5 +6,6 @@ module.exports = input => ({
   id: uuid.v4(),
   destination: createDestination(),
   expressionGroup: createExpressionGroup(),
+  operator: "And",
```
[the remark this answers is not in the document]
Comment (heofs) [reply eq-author-api/src/businessLogic/createRoutingRule.js]: I have added the constant now.

File: eq-author-api/migrations/addExpressionOperator.js (hunk 1 above)
Comment (heofs) [reply eq-author-api/migrations/addExpressionOperator.js]: I have added the constant now.

**Pull request merged**